sdk/core/Azure.Core.TestFramework/src/RecordSession.cs

```diff
@@ -99,21 +99,6 @@ public void Sanitize(RecordedTestSanitizer sanitizer)
             }
         }
 
-        public bool IsEquivalent(RecordSession session, RecordMatcher matcher)
-        {
-            if (session == null)
-            {
-                return false;
-            }
-
-            // The DateTimeOffsetNow variable is updated any time it's used so
-            // we only care that both sessions use it or both sessions don't.
-            var now = TestRecording.DateTimeOffsetNowVariableKey;
-            return session.Variables.TryGetValue(now, out string _) == Variables.TryGetValue(now, out string _) &&
```

Comment:
You also need remove EntryEquivalentComparer and all leftovers from RecordMatcher

Reply:
And from every library that was overriding them

Reply:
Deleting code
![](https://media.giphy.com/media/LmNwrBhejkK9EFP504/giphy.gif)